viz.yaml

```diff
@@ -179,9 +179,10 @@ publish:
       topojson: "topojson"
       header_css: "lib-header-css"
       footer_css: "lib-footer-css"
+      content_css: "lib-content-css"
       map_css: "map_css"
-      main_css: "main_css"
-      social: "social_section"
+      custom_css: "custom_css"
+      social: "social-section"
```

Comment:
i just asked lindsay to rename `social-section` to `social_section`, so i think this line needs to be edited to an underscore now